src/recaptcha.js

```diff
@@ -10,7 +10,7 @@ const ReCAPTCHA = React.createClass({
     type: PropTypes.oneOf(["image", "audio"]),
     tabindex: PropTypes.number,
     onExpired: PropTypes.func,
-    size: PropTypes.oneOf(["compact", "normal"]),
+    size: PropTypes.oneOf(["compact", "normal", "invisible"]),
```

Comment:
@albybarber the name is `type` because that's the name from google documentation. https://developers.google.com/recaptcha/docs/display#render_param

![image](https://cloud.githubusercontent.com/assets/4422516/23974882/f00c0576-099a-11e7-9951-66257b953726.png)


I feel we should keep it under `size` as it was done because that's still the property name in the official doc. Though I can see the confusion with the fact that they have 2 different "type".

![image](https://cloud.githubusercontent.com/assets/4422516/23974874/e37dfaee-099a-11e7-8db7-e88e8ef9f5b7.png)